src/content/docs/logs/ui-data/query-time-parsing.mdx

```diff
@@ -0,0 +1,412 @@
+            <td>
+                Timing
+            </td>
+            <td>
+                Applied at ingest
+            </td>
+            <td>
+                Applied when you query
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Results
+            </td>
+            <td>
+                Makes permanent changes to stored log 
+            </td>
+            <td>
+                Temporarily alters the logs table in the UI
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Live tail logs
+            </td>
+            <td>
+                Live tail logs include any extracted values from ingest parsing
+            </td>
+            <td>
+                Live tail logs **don't** include any extracted values from query time parsing
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Exported logs
+            </td>
+            <td>
+                Exported logs include any extracted values from ingest parsing
+            </td>
+            <td>
+                Exported logs **don't** include any extracted values from query time parsing
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Number of attributes
+            </td>
+            <td>
+                A maximum of [255 attributes](/docs/logs/log-api/log-event-data/#events) is available at ingest (the actual number attributes you can parse at ingest depends on the nature of your logs)
+            </td>
+            <td>
+                You can parse a maximum of 32 temporary attributes across all rules for query time parsing
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## How to create a query time parsing rule [#how-to] 
+
+Here's an overview of the tasks for creating query time parsing rules:
+
+<Steps>
+    <Step>
+        #### Select attribute value to parse [#select-attribute]
+        You can start creating a query time parsing rule by selecting an attribute to parse:
+
+            1. In the log table or in the **Formatted** tab of the log details view, highlight an attribute value you'd like to parse, known as the anchor string. Keep the following in mind:
+                * Your highlighted text should include the value(s) you want to extract and the surrounding string characters that will help identify the location of the extracted value(s). 
+                * If your initial anchor string is anywhere in the middle of the original attribute value, include at least one character before and after the values you want to extract.
+                * If you're highlighting the entire attribute value, you don't need to worry about characters before and after the values you're extracting.
+        <Callout variant="tip">
+            You cannot highlight blob values to parse.
+        </Callout>
+            2. Click the **Create query time parsing rule** option. 
+
+                **Log table:**
+                <img
```

Comment:
Can you add an arrow to this visual? The green box blends so well into the UI it's difficult to see.

Reply:
@homelessbirds yes, I will add a green arrow from the log record to the menu.